lms/migrations/versions/73f0011260e4_usage_report.py

```diff
@@ -1,15 +1,15 @@
 """Create the usage_report table.
 
 Revision ID: 73f0011260e4
-Revises: 329313b38de1
+Revises: b512a5cf64ed
 """
 
 import sqlalchemy as sa
 from alembic import op
 from sqlalchemy.dialects import postgresql
 
 revision = "73f0011260e4"
-down_revision = "329313b38de1"
+down_revision = "b512a5cf64ed"
```

Comment:
Another migration was merged before I merged this stream of work one.

Updating the parent to keep a linear migration history.